ModInstances/SpookyWorld.cs

```diff
@@ -112,105 +106,90 @@ public override void PostDrawTiles()
+                        var pickUp = "Right click to pick up";
+                        var g = Main.fontMouseText.MeasureString(pickUp);
                         // Main.spriteBatch.DrawString(Main.fontMouseText, pickUp, Main.MouseScreen + new Vector2(25, 25), Color.White, 0f, new Vector2(0, 0), Main.essScale * 5/*1f*/, SpriteEffects.None, 0f);
                         Main.spriteBatch.DrawString(Main.fontMouseText, pickUp, Main.MouseWorld + new Vector2(15, -35) - Main.screenPosition, Color.White * lighting, 0f, new Vector2(g.X / 2, 5), 1f, SpriteEffects.None, 0f);
                     }
-                }
-            }
+
             Main.spriteBatch.End();
         }
+
         public override void PostUpdate()
         {
-            ModNPC modNPC = new ModNPC();
-            if (modNPC.npc.townNPC)
-            {
-                modNPC.CanTownNPCSpawn(999, int.MaxValue);
-            }
-            for (int index = 0; index < Main.maxNPCs; index++)
+            var modNPC = new ModNPC();
+            if (modNPC.npc.townNPC) modNPC.CanTownNPCSpawn(999, int.MaxValue);
+
+            for (var index = 0; index < Main.maxNPCs; index++)
             {
-                NPC npc = Main.npc[index];
+                var npc = Main.npc[index];
                 if (npc.type == ModContent.NPCType<Stalker>() && npc.active)
-                {
                     Main.player[Main.myPlayer].GetModPlayer<SpookyPlayer>().stalkerConditionMet = true;
-                }
             }
         }
-		public override void ModifyWorldGenTasks(List<GenPass> tasks, ref float totalWeight)
-		{
-			int shiniesIndex = tasks.FindIndex(genpass => genpass.Name.Equals("Planting Trees"));
-
-			if (shiniesIndex != -1)
-			{
-				tasks.Insert(shiniesIndex + 1, new PassLegacy("Planting Pages", PagesOnTrees));
-			}
-		}
-		private void PagesOnTrees(GenerationProgress progress)
-		{
-            int numOfPages = 8; // Change this ;)
-			progress.Message = "Randomly slapping pages on trees...";
-
-            List<Point> rightSideSpots = new List<Point>();
-            List<Point> leftSideSpots = new List<Point>();
-
-            for (int i = 2; i < Main.maxTilesX - 2; i++)
-			{
-                for (int j = 2; j < Main.maxTilesY - 2; j++)
-                {
-                    Tile leftTile = Main.tile[i - 1, j];
-                    Tile leftMostTile = Main.tile[i - 2, j];
-                    Tile rightTile = Main.tile[i + 1, j];
-                    Tile rightMostTile = Main.tile[i + 2, j];
-
-                    if (leftTile.type == TileID.Trees && !leftMostTile.active() && leftTile.frameX == 0 && leftTile.frameY < 66)
-					{
-                        leftSideSpots.Add(new Point(i, j));
-					}
-                    else if (rightTile.type == TileID.Trees && !rightMostTile.active() && rightTile.frameX == 0 && rightTile.frameY < 66)
-					{
-                        rightSideSpots.Add(new Point(i, j));
-					}
-                }
+
```

Comment:
Idfk what this was, github fucking broke the indentations here, they don’t exist on my build

Reply:
Hmm yeah. Github likes to do that